.github/ISSUE_TEMPLATE/support_request.md

```diff
@@ -0,0 +1,4 @@
+---
+name: ⛔️ Support Question
+description: See https://forum.sentry.io/ for support requests.
```

Comment:
Can we change this to https://sentry.io/support/ instead. That page will automatically redirect them, and if they're a saas user it might provide alternative options.

Reply:
:+1: Done.